src/main/webapp/WEB-INF/jsp/login/invalidStateCookieWarning.jsp

```diff
@@ -67,14 +67,22 @@
   <br>
   <div class="description">
     <span class="icon icon-info blue"></span>
-    <fmt:message key="bookmark_link_1" bundle="${resword}"/>
-    <a href="/OpenClinica/MainMenu">link</a>
-    <fmt:message key="bookmark_link_2" bundle="${resword}"/>
+    <fmt:message key="bookmark_link" bundle="${resword}"/>
+    <a href="/OpenClinica/MainMenu"><p id="url"></p></a>
   </div>
 </div>
 
 <jsp:include page="../include/footer.jsp"/>
 
+<script>
+  $(document).ready(function(){
+    var url = window.location.href;
+    var n = url.indexOf("/OpenClinica/InvalidStateCookieWarning");
+    url = url.substring(0, n) + "/OpenClinica/MainMenu";
+    document.getElementById('url').innerHTML = url;
```

Comment:
The mockup shows only the base url and does not have "/OpenClinica/MainMenu" at the end